From 505c8146fc47029454cfa060e6a914391e21e1a5 Mon Sep 17 00:00:00 2001
From: Andreas Scherer <andreas_github@freenet.de>
Date: Mon, 8 Aug 2016 19:16:19 +0200
Subject: [PATCH 6/6] Update information about documentation.

The toc pages were missing. MMIX support is provided via Germany.
---
 Makefile |  6 +++---
 README   | 13 +++++++------
 2 files changed, 10 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index 9cd6a27..8c67def 100644
--- a/Makefile
+++ b/Makefile
@@ -65,13 +65,13 @@ ALL = $(WEBFILES) $(TESTFILES) $(MISCFILES)
 basic:  mmixal mmix
 
 doc:    mmix-doc.ps mmixal.dvi mmix-sim.dvi
-	dvips -n13 mmixal.dvi -o mmixal-intro.ps
-	dvips -n8 mmix-sim.dvi -o mmix-sim-intro.ps
+	dvips -pp 0-13 mmixal.dvi -o mmixal-intro.ps
+	dvips -pp 0-8 mmix-sim.dvi -o mmix-sim-intro.ps
 
 all:    mmixal mmix mmotype mmmix
 
 clean:
-	rm -f *~ *.o *.c *.h *.tex *.log *.dvi *.toc *.idx *.scn *.ps core
+	rm -f *~ *.o *.c *.h *.tex *.log *.dvi *.toc *.idx *.scn *.ps *.pdf core
 
 mmix-pipe.o: mmix-pipe.c abstime
 	./abstime > abstime.h
diff --git a/README b/README
index e754a51..9d3f29d 100644
--- a/README
+++ b/README
@@ -67,12 +67,13 @@ makefile.dos      DOS makefile (for DOS/Windows platforms)
 HOW TO GET STARTED
 
 First try "make doc"; this, at least on UNIX-like systems equipped with dvips,
-should create three PostScript documents called mmix-doc.ps, mmixal.ps, and
-mmix-sim.ps. Print these files, and move their tables-of-contents
-pages to the front (those pages appear at the end). Read the introductory
-parts of those documents, and please report any errors you find to
-knuth-bug@cs.stanford.edu. If your system is unable to create those documents
-for some reason, go to
+should create three PostScript documents called mmix-doc.ps, mmixal-intro.ps,
+and mmix-sim-intro.ps. Print these files, and move each table-of-contents
+page to the front (those pages appear at the end). Read those documents,
+and please report any errors you find to ruckert@cs.hm.edu.
+If your system is unable to create those documents for some reason, go to
+   http://mmix.cs.hm.edu/doc
+for pdf files, or to
    http://www-cs-faculty.stanford.edu/~knuth/mmix-news.html
 and download the compressed PostScript files from there.
 Say "make clean" when you've got hardcopy.
-- 
2.9.2

